hnn_core/cell.py

```diff
@@ -239,6 +242,13 @@ def record_current_soma(self):
                 " but no self.synapses dict was found")
             pass
 
+    def record_voltage_soma(self):
+        """Record current at soma."""
+        if self.record_vsoma:
```

Comment:
is it necessary to expose this attribute to the user?

Reply:
@jasmainak are you critiquing the scope of this attribute or that the `record_vsoma` arg set in the constructor is set as an attribute rather just directly passed to `_Cell.record_voltage_soma()`?

Reply:
I was thinking it could be a private attribute. But on second thoughts, I think it can stay as it is ... at least for now